src/robot/output/console/__init__.py

```diff
@@ -20,16 +20,20 @@
 from .verbose import VerboseOutput
 
 
-def ConsoleOutput(type='verbose', width=78, colors='AUTO', markers='AUTO',
+def ConsoleOutput(type='normal', width=78, colors='AUTO', markers='AUTO',
                   stdout=None, stderr=None):
     upper = type.upper()
-    if upper == 'VERBOSE':
+    if upper in 'NORMAL':
```

Comment:
Yes, it is a typo. I was using before the commit "in ('NORMAL', 'VERBOSE')" and a second if inside it. I changed it to two ifs and forgot to change this part to use '==', during the time I was doing the commit.